Presentations/ElevatorPitch.md

```diff
@@ -0,0 +1,17 @@
+# Elevator Pitch Outline
+## Who
+- Be A Ruby local charity
+- distribute food and clothing to low income communities
+## Why
+- track trailer inventory like cups, to go boxes, and silverware
```

Comment:
What about adding something about why they want to track inventory? Something about how they don't currently have a system and would like one to keep organized and know when they are running out of a specific item